rocketpy/tools.py

```diff
@@ -381,6 +382,107 @@ def check_requirement_version(module_name, version):
     return True
 
 
+def is_float(element):
+    """
+    Returns a boolean indicating us if an element is convertible to a float or not.
+    True : the element is convertible to a float
+    False : the element is not convertible to a float
+
+    Parameters
+    ----------
+    element : any
+        This is the element to test.
+
+    Returns
+    -------
+    result : boolean
+        The element is convertible or not.
+    """
+    if element is None:
+        return False
+    try:
+        float(element)
+        return True
+    except ValueError:
+        return False
+
+
+def return_first_data(source):
+    """
+    Returns the first data of a CSV file.
+
+    Parameters
+    ----------
+    source : string
+        This is the file path to the csv.
+
+    Returns
+    -------
+    result : any
+        The first data of the CSV file.
+    """
+    native_data = open(source)
```

Comment:
I recommend using a "with" statement, so you ensure that the file always get closed.

https://realpython.com/why-close-file-python/

Reply:
Done!